src/relay/qnn/op/requantize.cc

```diff
@@ -256,13 +256,20 @@ Expr RequantizeQnnCanonicalize(const Attrs& attrs, const Array<Expr>& new_args,
  */
 bool RequantizeRel(const Array<Type>& types, int num_inputs, const Attrs& attrs,
                    const TypeReporter& reporter) {
+  // Expected Types: data, input_scale, input_zero_point, output_scale, output_zero_point, output
   ICHECK_EQ(types.size(), 6);
   const auto* data = types[0].as<TensorTypeNode>();
 
   if (data == nullptr) {
     return false;
   }
 
+  // Check the scale and zero point types
+  for (size_t i = 3; i < 5; ++i) {
```

Comment:
These values also dont seem to line up with expected types.

Reply:
A few lines down you'll see assignments for input scale and input zero point, but checks on output_scale and output_zero_point

Reply:
So this is pre-checking output_scale and output_zero_point